django_project/frontend/src/pages/OrganisationInformation/index.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useState } from "react";
+import Helmet from "react-helmet";
+import {
+  Box,
+  Heading,
+  Flex,
+  Input,
+  Table,
+  Tbody,
+  Tr,
+  Td,
+  Thead,
+  Button,
+  Badge,
+  IconButton,
+  Tabs,
+  TabList,
+  Tab,
+  TabPanels,
+  TabPanel,
+  Divider,
+} from "@chakra-ui/react";
+import { FaPlus, FaTrash } from "react-icons/fa";
+import Header from "../../components/Header";
+import Sidebar1 from "../../components/SideBar";
+import '../../styles/index.css';
+
+// Define types for the data
+interface Member {
+  user: string;
+  role: string;
+}
+
+interface Invitation {
+  user: string;
+  role: string;
+  status: string;
+}
+
+interface Organization {
+  members: Member[];
+  invitations: Invitation[];
+}
+
+// Dummy data for demonstration
+const organizations: { [key: string]: Organization } = {
+  default: {
+    members: [],
+    invitations: [],
+  },
+  org2: {
+    members: [],
+    invitations: [],
+  },
+};
+
+export default function OrganisationInformation() {
+  const [isSidebarOpen, setSidebarOpen] = useState(true);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  return (
+    <>
+      <Helmet>
+        <title>Organisation Information - Manage Organisation Details</title>
+        <meta name="description" content="Manage your organisation's information, members, and invitations." />
+      </Helmet>
+      <Header />
+
+      <Box bg="white" w="100%">
+        <Flex direction={{ base: "column", md: "row" }} gap="30px" alignItems="start">
+          {/* Sidebar */}
+          <Sidebar1 display={{ base: "none", md: "flex" }} />
+
+          {/* Main Content */}
+          <Box flex="1" ml={{ base: "55px", md: "0px" }} mt={{ base: "0px", md: "20px" }} width={{base: "80%", md: "auto"}} overflow={'auto'}>
+            <Heading size="lg" mb={6} color="black">
+              My Organisations
+            </Heading>
+
+            {/* Organisation Tabs */}
+            <Tabs variant="unstyled">
+              <Box width="fit-content" border="1px solid" borderColor="gray.300" borderBottom="none" borderRadius="5px" overflow="hidden">
+                <TabList>
+                  {Object.keys(organizations).map((orgKey, idx) => (
+                    <Tab
+                      key={idx}
+                      _selected={{ color: "white", bg: "dark_green.800" }}
+                      _hover={{ bg: "light_green.400" }}
+                      px={6}
+                      py={2}
+                    >
+                      {orgKey}
+                    </Tab>
+                  ))}
+                </TabList>
+              </Box>
+
+              <Divider mb={6} borderColor="black" borderWidth="1px" width={{base:"auto", md:"99%"}}/>
+
+              <TabPanels>
+                {Object.values(organizations).map((organization, index) => (
```

Comment:
let's use state for organizations, so when we have real data it's going to be easier to implement.